data/transit/route/light_rail.json

```diff
@@ -148,7 +148,12 @@
       "locationSet": {"include": ["001"]},
       "tags": {
         "network": "Verkehrs- und Tarifverbund Stuttgart",
+        "network:guid": "DE-BW-VVS",
+        "network:wikidata": "Q2516108",
+        "network:wikipedia": "de:Verkehrs- und Tarifverbund Stuttgart",
```

Comment:
Or should this rather link to the English Wikipedia page? (I copied these three tags from `bus.json`.)